test/Fail/RecordWhere2.err

```diff
@@ -0,0 +1,3 @@
+error: [UnsolvedMetaVariables]
+Unsolved metas at the following locations:
+  RecordWhere2.agda:5,7-7,12
```

Comment:
I'm guessing the "The record [whatever] does not have field `distraction` but it would have field `proj₁`" warning got suppressed or something? Can we get a test case ensuring that it happens?

Reply:
> and may include other definitions than the fields of the record, including opening of modules.

Ah, nevermind! Though in that case I think we should have a better warning for the missing field than an unsolved meta...?

Reply:
The `TooManyFields` warning is indeed turned off for `record where`, because one of the main features of the construct is that you can define helper functions that are not fields and that shouldn't give you a warning. There are several examples of this in the `RecordWhere` test case in `test/Succeed`.